open-company/open-company-web
# Mobile WST

Card: https://trello.com/c/fQSjPQm4

To test:
- open the app on mobile
- click on a few posts WST label
- [x] do you see the read list appear? Good
- [x] do you see also the unread list? Good
- [x] can you send reminders? Good
- [x] can you use the search? Good
